presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -653,12 +610,27 @@ protected RowExpression visitTryExpression(TryExpression node, Void context)
             throw new UnsupportedOperationException("Must desugar TryExpression before translate it into RowExpression");
         }
 
+        private RowExpression buildEquals(RowExpression lhs, RowExpression rhs)
+        {
+            return call(EQUAL.getOperator(),
+                    functionResolution.comparisonFunction(ComparisonExpression.Operator.EQUAL, lhs.getType(), rhs.getType()),
+                    BOOLEAN,
+                    lhs,
+                    rhs);
+        }
+
         @Override
         protected RowExpression visitInPredicate(InPredicate node, Void context)
         {
             ImmutableList.Builder<RowExpression> arguments = ImmutableList.builder();
-            arguments.add(process(node.getValue(), context));
+            RowExpression lhs = process(node.getValue(), context);
```

Comment:
`lhs` -> `value`

Reply:
Done. What can I say? I'm compiler guy and lhs and rhs are the only way to refer to expression operands! :)